### PR TITLE
OGM-1398 Move slot properties

### PR DESCRIPTION
**Pull request by DavideD**
https://hibernate.atlassian.net/browse/OGM-1398

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -69,6 +69,9 @@
         <version.wildfly>11.0.0.Final</version.wildfly>
         <hibernateWildflyClassifier>wildfly-11-dist</hibernateWildflyClassifier>
 
+        <!-- This slot property is used by the modules porject and the integration tests -->
```
Comment (Sanne) [inline pom.xml]: typo ;)

Comment (Sanne) [reply pom.xml]: I'll fix it myself and merge if the tests are good.

**Comment by DavideD** [comment]
Damn :)

**Comment by Sanne** [comment]
merged, thanks!

**Pull request closed without merging**